modules/nw-egress-ips-about.adoc

```diff
@@ -18,6 +18,11 @@ The {product-title} egress IP address functionality allows you to ensure that th
 For example, you might have a pod that periodically queries a database that is hosted on a server outside of your cluster. To enforce access requirements for the server, a packet filtering device is configured to allow traffic only from specific IP addresses.
 To ensure that you can reliably allow access to the server from only that specific pod, you can configure a specific egress IP address for the pod that makes the requests to the server.
 
+[NOTE]
+====
+The {rh-openstack} egress IP address feature creates a neutron reservation port called `egressip-<IP address>`. You can assign a floating IP address to this reservation port to have a predictable SNAT address for egress traffic going outside of the cluster. When an egress IP address on an {rh-openstack} network is moved from one node to another, because of a node failover for example, the neutron reservation port is removed and recreated. This means that the floating IP association will be lost and you need to manually reassign the floating IP address to the new reservation port.
```

Comment:
1. As per my understanding `egress traffic` is what goes outside the cluster. If `going outside of the cluster` is not needed, it can be removed.
2. `for example` - may be a preceding `,` ?
3. `the floating IP association will be lost` - IMO, this can be reworded to have present tense.